setup.py

```diff
@@ -19,7 +19,7 @@
     install_requires=[
         "toolz>=0.9.0,<1.0.0;implementation_name=='pypy'",
         "cytoolz>=0.9.0,<1.0.0;implementation_name=='cpython'",
-        "eth-abi>=1.1.1,<2",
+        "eth-abi>=2.0.0a1,<3.0.0",
```

Comment:
I think it's a must to support old versions of eth-abi. Once we get these new filters into the wild and we're happy with the results after seeing people try to use them for a bit, maybe it will be time to focus on getting v5 out the door.